crates/astria-composer/README.md

```diff
@@ -0,0 +1,27 @@
+# Astria Composer
+
+Composer sits between a rollup execution node and the Astria Shared Sequencer to submit transactions. As currently implmented, it follows a websocket on a Geth Node for new transactions, wraps and signs the bytes into a sequencer transaction.
```

Comment:
specify the relationship (n-1, n-n) between the execution nodes and composer? from what I understand there can be many composers and each composer can handle many rollup nodes